config/app_config.cfg

```diff
@@ -0,0 +1,16 @@
+[GLOBAL]
+NUM_DP_SELECTION_RULES = 2
+
+[DP_SELECTION_RULE_1]
+DPID = 1
+DPNAME = "menlo"
+MCC = "111"
+MNC = "221"
+ENB_ADDRESS = "1.1.1.2"
+
+[DP_SELECTION_RULE_2]
+DPID = 2
+DPNAME = "tucsan"
+MCC = "112"
+MNC = "222"
+ENB_ADDRESS = "2.2.2.1"
```

Comment:
Only 1 ENB per site?

Referencing back to our conversation, this seems painful to provide DP details using config files, instead of over the network. Do you agree?

Reply:
How come (list of) TAC not here?

Reply:
In the first version I am adding only MCC/MNC. I will add list of TAIs as well. Since code is already in code review, let me add TAI list as well.

Reply:
we should not need ENB_ADDRESS. right?

Reply:
I was thinking about how to identity dataplane...Then i see that after s1 Release, if UE does service request then then we can use ENB address also as DP selection. Or we can continue to use the same DP (whatevery was selected during initial attach)... As of now eNB address is not used.

Reply:
Not sure i understand. Looking at create session request I do not see eNB IP, so not sure how this will help identify.

Reply:
Correct. CSReq does not have eNB address. But lets see following cases
1. CSReq comes in. One of the edge is selected. S1 release. UE goes in idle mode. Logically we deleted the session from the DP as well. Now if MBReq ( in response to paging) comes in the we can reselect DP. This is where enb address can be used.
2. x2 Handover case. SGW change. In this case CSReq comes with eNB address.

Reply:
How are we handling these today? if we are not, can we separate the PRs?

Reply:
MCC, MNC and TAC will make it TAI. I will add TAC in the list. I will remove ENB_ADDRESS. Since its causing confusion. Let each DP report eNB address to CP. At least for now I am keeping ENB address outside.